openmetadata-ui/src/main/resources/ui/src/utils/EntityLineageUtils.tsx

```diff
@@ -96,15 +96,9 @@ export const getHeaderLabel = (
   );
 };
 
-export const onLoad = (
-  reactFlowInstance: ReactFlowInstance,
-  length?: number,
-  forceZoomReset = false
-) => {
+export const onLoad = (reactFlowInstance: ReactFlowInstance) => {
   reactFlowInstance.fitView();
-  if (forceZoomReset || (length && length <= 2)) {
-    reactFlowInstance.zoomTo(zoomValue);
-  }
+  reactFlowInstance.zoomTo(zoomValue);
```

Comment:
From where we are getting this `zoomValue` it should be constant or via function params

Reply:
it's from constant!